README.md

```diff
@@ -2,43 +2,47 @@
 
 This is a golang integration for Pyroscope — open source continuous profiling platform.
 
-For more information, please visit our [golang integration documentation](https://pyroscope.io/docs/golang).
+For more information, please visit our [golang integration documentation](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/go_push/).
 
 ### Profiling Go applications
+
 To start profiling a Go application, you need to include our go module in your app:
+
 ```
-# make sure you also upgrade pyroscope server to version 0.3.1 or higher
-go get github.com/pyroscope-io/client/pyroscope
+go get github.com/grafana/pyroscope-golang/profiler
 ```
 
 Then add the following code to your application:
 
 ```go
 package main
 
-import "github.com/pyroscope-io/client/pyroscope"
+import "github.com/grafana/pyroscope-golang/profiler"
 
 func main() {
-  pyroscope.Start(pyroscope.Config{
+  profiler.Start(profiler.Config{
```

Comment:
I'd like to propose a slightly different approach – to move everything user need to import to the root and change the root package name to "pyroscope": https://github.com/grafana/pyroscope-golang/compare/main...chore/reorganize-package. In this way users won't need to change their code (see [`example`](https://github.com/grafana/pyroscope-golang/compare/main...chore/reorganize-package#:~:text=3-,example/main.go,-%40%40%20%2D3%2C11%20%2B3%2C12%20%40%40%20package)).

Also, in the branch the module is called `pyroscope-go`, which in my opinion is more correct (as the official name is Go) and commonly used:
 - https://github.com/kubernetes/client-go
 - https://github.com/elastic/go-elasticsearch
 - https://github.com/opensearch-project/opensearch-go
 - https://github.com/confluentinc/confluent-kafka-go
 - https://github.com/segmentio/parquet-go

Although, that would require a change of the repo name

Reply:
I think we could make the repo name change ?

Reply:
let me prepare everything and see.